lib/portage/dispatch_conf.py

```diff
@@ -40,7 +40,7 @@ def diffstatusoutput(cmd, file1, file2):
     args = (portage._unicode_encode(x, errors="strict") for x in args)
     proc = subprocess.Popen(args, stdout=subprocess.PIPE, stderr=subprocess.STDOUT)
     output = portage._unicode_decode(proc.communicate()[0])
-    if output and output[-1] == "\n":
+    if output and output.endswith("\n"):
```

Comment:
Not really related to the rest of the patch, but I'll allow it.

Reply:
I took it out anyway!

It's in the Miscellaneous patch now. 😎